src/builtin/tables.scrbl

```diff
@@ -138,20 +263,19 @@ end
 
   @section{Ordering Tables}
 
-Since a table consists of a sequence of rows, one may desire to arrange
-those rows in some particular order. This can be done with any column whose
-type supports the use of @pyret{<} and @pyret{>} by using an @pyret{order}
-expression:
+To arrange the rows of a table in some particular order, use an @pyret{order}
+expression.  This can be done with any column whose
+type supports the use of @pyret{<} and @pyret{>}, including @pyret{String}s. 
```

Comment:
Should we just assume both your changes and the doc changes are being pushed shortly before Colorado?